client/app/telecom/telephony/line/phone/reboot/telecom-telephony-line-phone-reboot.controller.js

```diff
@@ -7,7 +7,7 @@ angular.module('managerApp').controller('TelecomTelephonyLinePhoneRebootCtrl', f
       .getGroup($stateParams.billingAccount)
       .then(group => group.getLine($stateParams.serviceName).getPhone()).then((phone) => {
         self.phone = phone;
-        self.isRebootable = /^phone\.(thomson|swissvoice)/.test(self.phone.brand);
+        self.isRebootable = !/^phone\.(gigaset|cisco.spa112)/.test(self.phone.brand);
```

Comment:
Could you please export this is in a constant ? :)
This would make it easier to maintain (we wouldn't have to look for the values :))

Reply:
@Steffy29 In that case and according the @FredericEspiau feedback, don't hesitate to create a ticket associated with this suggestion.

In that way it could be addressed later on.

Since this Pull request is now in "Quality check OK" status, I will merge it.

Thanks!